polymer/simple-wrapper-element/web/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+
+<!--
+  Copyright (c) 2015, the Dart project authors.
+  All rights reserved. Use of this source code
+  is governed by a BSD-style license that can be
+  found in the LICENSE file.
+-->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Simple Wrapper Element</title>
+
+    <script async src="packages/web_components/webcomponents-lite.js"></script>
+    <script type="application/dart" src="index.dart"></script>
```

Comment:
Shouldn't these be async? Also, the stagehand web-simple template moves the type to later, which I like because it avoids unnecessary visual diffs:

    <script async src="main.dart" type="application/dart"></script>
    <script async src="packages/browser/dart.js"></script>